.github/workflows/publish.yml

```diff
@@ -33,8 +33,10 @@ jobs:
           # from refs/tags/v1.2.3 get 1.2.3
           VERSION=$(echo $GITHUB_REF | sed 's#.*/v##')
           PLACEHOLDER="__version__\ =\ \"0.0.0+dev0\""
-          grep "$PLACEHOLDER" "unionml/__init__.py"
-          sed -i "s#$PLACEHOLDER#__version__ = \"$VERSION\"#g" "unionml/__init__.py"
+          # grep "$PLACEHOLDER" "unionml/__init__.py"
+          # sed -i "s#$PLACEHOLDER#__version__ = \"$VERSION\"#g" "unionml/__init__.py"
+          grep "$PLACEHOLDER" "setup.py"
+          sed -i "s#$PLACEHOLDER#__version__ = \"$VERSION\"#g" "setup.py"
```

Comment:
Looking forward to your comments on this.

cc: @cosmicBboy

Reply:
is the `sed` replacement in `unionml/__init__.py` still necessary after your changes?

Reply:
Made changes now to to only use `sed` once to apply to `setup.py`.